SUIApp/SUIApp/BasketView.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  BasketView.swift
+//  SUIApp
+//
+//  Created by Vladka on 22.01.2023.
+//
+
+import SwiftUI
+
+/// Экран корзины
+struct BasketView: View {
+    // MARK: - Private Constants
+    
+    private enum Constants {
+        static let currentTariffText = "Текущий тариф"
+        static let nameTitleText = "Название:"
+        static let periodTitleText = "Период:"
+        static let priceTitleText = "Стоимость:"
+        static let dayText = "дней"
+        static let dollarSymbolText = "$"
+        static let editTariffText = "Изменить тариф?"
+        static let basketTitleText = "Корзина"
+        static let buyText = "Купить"
+        static let alertTitleText = "Покупка совершена!"
+        static let oneNumber = 1
+        static let cornerRadiusRectangleNumber: CGFloat = 20
+        static let defaultSpacingNumber: CGFloat = 150
+    }
+    
+    // MARK: - Public Property
+    
+    var body: some View {
+        NavigationView {
+            ZStack {
+                RoundedRectangle(cornerRadius: Constants.cornerRadiusRectangleNumber)
+                    .fill(.yellow)
+                VStack {
+                    Text(Constants.currentTariffText)
+                        .font(.title)
+                    Spacer()
+                    HStack(spacing: Constants.defaultSpacingNumber) {
+                        VStack(alignment: .leading) {
+                            Text(Constants.nameTitleText)
+                            Text(Constants.periodTitleText)
+                            Text(Constants.priceTitleText)
+                            
+                        }
+                        .bold()
+                        VStack(alignment: .trailing) {
+                            Text(purchaseModel.planeName)
+                            Text("\(purchaseModel.planePeriod) \(Constants.dayText)")
+                            Text("\(purchaseModel.planePrice)\(Constants.dollarSymbolText)")
+                        }
+                    }
+                    Spacer()
+                    Button(Constants.editTariffText) {
+                        tabSelection = Constants.oneNumber
+                    }
+                    Button(Constants.buyText) {
+                        isAlertShown.toggle()
+                    }.alert(isPresented: $isAlertShown) {
```

Comment:
```suggestion
                    }
                    .alert(isPresented: $isAlertShown) {
```
модификаторы с новой строки